ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -180,6 +198,10 @@ export class PrepareMigrationComponent implements OnInit {
             name: 'Minimal downtime Migration',
             value: MigrationTypes.lowDowntimeMigration,
           },
+          {
```

Comment:
Add this conditionally depending on source database

Reply:
Added check to show dataproc migration mode only for MySQL
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/250c1607ededfd68eee6f3cf324b820a82699f33